### PR TITLE
Schema examples for BC5CDR and BioASQ9b updated

### PR DESCRIPTION
**Pull request by jason-fries**
This fully implements BC5CDR and BioASQ9b in the source (original) and bigbio schemas as worked examples for the hackathon.

### CODE REVIEW

File: examples/bc5cdr.py [hunk 1]
```diff
@@ -58,65 +65,110 @@
...
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "document_id": datasets.Value("string"),
+                    "passages": [
+                        {
+                            "id": datasets.Value("string"),
+                            "type": datasets.Value("string"),
+                            "text": datasets.Value("string"),
+                            "offsets": datasets.Sequence([datasets.Value("int32")]),
```
Comment (galtay) [inline examples/bc5cdr.py:141]:
```suggestion
                            "offsets": datasets.Sequence(datasets.Value("int32")),
```

File: examples/bc5cdr.py [hunk 2]
```diff
@@ -183,45 +227,99 @@ def _split_generators(self, dl_manager):
...
+                    "passages": [],
+                    "entities": [],
+                    "relations": [],
+                }
+                uid += 1
+
+                char_start = 0
+                # passages must not overlap and spans must cover the entire document
+                for passage in xdoc.passages:
+                    offsets = [[char_start, char_start + len(passage.text)]]
```
Comment (galtay) [inline examples/bc5cdr.py:293]: don't need nested iterables for passage offsets and its incompatible with the single string field for `text`

**Review by galtay — CHANGES_REQUESTED** [review]
Nice example! Looks like there is just one offset iterable change to make and then LGTM

File: examples/bc5cdr.py (hunk 1 above)
Comment (jason-fries) [reply examples/bc5cdr.py:141]: Using`datasets.Sequence([datasets.Value("int32")])` for consistency with entities, etc, per slack conversation

File: examples/bc5cdr.py (hunk 2 above)
Comment (jason-fries) [reply examples/bc5cdr.py:293]: Again. matching for consistency per Slack

**Pull request merged**